drivers/kscan/kscan_ft5336.c

```diff
@@ -179,18 +184,36 @@ static int ft5336_init(const struct device *dev)
 {
 	const struct ft5336_config *config = dev->config;
 	struct ft5336_data *data = dev->data;
+	int r;
+
+	(void) r;
 
 	if (!device_is_ready(config->bus.bus)) {
 		LOG_ERR("I2C controller device not ready");
 		return -ENODEV;
 	}
 
+#ifdef CONFIG_KSCAN_FT5336_POWER_GPIO
+
+	if (!device_is_ready(config->power_gpio.port)) {
+		LOG_ERR("power_gpio is not ready");
+		return -ENODEV;
+	}
+
+	r = gpio_pin_configure_dt(&(config->power_gpio),
+			GPIO_OUTPUT | GPIO_ACTIVE_LOW | GPIO_OUTPUT_ACTIVE);
+	if (r < 0) {
+		LOG_ERR("Could not configure configure power GPIO pin");
+		return r;
+	}
+
+#endif
```

Comment:
you can use a regulator or power domains to achieve that without the need to modify the driver itself. See e.g. `boards/arm/thingy53_nrf5340/thingy53_nrf5340_common.dts`

Reply:
Interesting, that might also be useful for the display driver

Reply:
Done